staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types.go

```diff
@@ -38,7 +38,7 @@ type CustomResourceDefinitionSpec struct {
 	Group string `json:"group" protobuf:"bytes,1,opt,name=group"`
 	// Names are the names used to describe this custom resource
 	Names CustomResourceDefinitionNames `json:"names" protobuf:"bytes,3,opt,name=names"`
-	// Scope indicates whether this resource is cluster or namespace scoped.  Default is namespaced
+	// Scope indicates whether this resource is cluster or namespace scoped.
```

Comment:
I realized the godoc on this type doesn't follow our rules.  Can you file a follow up issue for 1.16 to get the godoc in the right naming form?

Reply:
https://github.com/kubernetes/kubernetes/issues/81519